### PR TITLE
dotCMS/core#20664 Apply design to suggestions

### PR DESCRIPTION
**Pull request by fmontes**
### Proposed Changes
- Add the design to the suggestions menu

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Review by rjvelazco — APPROVED** [review]

File: libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts [hunk 1]
```diff
@@ -15,28 +20,21 @@ import { SuggestionListComponent } from '../suggestion-list/suggestion-list.comp
...
-        type
-    }: {
-        payload?: DotCMSContentlet;
-        type: { name: string; level?: number };
-    }) => void;
+    @Input() onSelection: (props: SuggestionsCommandProps) => void;
     items: MenuItem[] = [];
 
-    constructor(private suggestionsService: SuggestionsService, private cd: ChangeDetectorRef) {}
+    title = 'Select a block';
```
Comment (alfredo-dotcms) [inline libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts:26]: i18n

File: libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts [hunk 2]
```diff
@@ -137,8 +198,10 @@ export class SuggestionsComponent implements OnInit {
                 take(1)
             )
             .subscribe((items) => {
+                this.title = 'Select a content type';
```
Comment (alfredo-dotcms) [inline libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts:201]: https://github.com/dotCMS/core-web/pull/1726/files#r695136574

File: libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts [hunk 3]
```diff
@@ -128,7 +187,9 @@ export class SuggestionsComponent implements OnInit {
                                             };
                                         });
 
+                                        this.title = 'Select a contentlet';
```
Comment (alfredo-dotcms) [inline libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts:190]: https://github.com/dotCMS/core-web/pull/1726/files#r695136574

File: libs/block-editor/src/lib/plugins/floating.plugin.ts [hunk 4]
```diff
@@ -78,15 +113,16 @@ export class FloatingActionsView {
...
-        if (composing || isSame) {
-            return;
-        }
-
+    /**
+     * Check the EditorState and based on that modify the DOM
+     *
+     * @param {EditorView} view
+     * @param {EditorState} prevState
+     * @return {*}
```
Comment (alfredo-dotcms) [inline libs/block-editor/src/lib/plugins/floating.plugin.ts:121]: remove?

File: libs/block-editor/src/lib/plugins/floating.plugin.ts [hunk 5]
```diff
@@ -1,71 +1,106 @@
...
+        });
+        this.editor.view.dispatch(transaction);
     };
 
-
-    createTooltip(options: Partial<Props> = {}) {
+    /**
+     * Create the tooltip for the element
+     *
+     * @param {Partial<Props>} [options={}]
```
Comment (alfredo-dotcms) [inline libs/block-editor/src/lib/plugins/floating.plugin.ts:100]: `options` not optional

File: libs/block-editor/src/lib/plugins/floating.plugin.ts [hunk 6]
```diff
@@ -117,23 +170,55 @@ export class FloatingActionsView {
...
+                    };
+                }
+
+                return {
+                    open: false
+                };
+            }
+        },
         props: {
             handleKeyDown(view: EditorView, event: KeyboardEvent) {
```
Comment (alfredo-dotcms) [inline libs/block-editor/src/lib/plugins/floating.plugin.ts:215]: doc?

File: libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts (hunk 1 above)
Comment (fmontes) [reply libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts:26]: Not suported yet

File: libs/block-editor/src/lib/plugins/floating.plugin.ts (hunk 5 above)
Comment (fmontes) [reply libs/block-editor/src/lib/plugins/floating.plugin.ts:100]: Not sure but this is how Document This generate it

Comment (alfredo-dotcms) [reply libs/block-editor/src/lib/plugins/floating.plugin.ts:100]: mmm weird
image

**Review by alfredo-dotcms — APPROVED** [review]

**Review by hmoreras — APPROVED** [review]

**Pull request merged**